src/test/unit_tests/jest_tests/test_nc_nsfs_connection_cli.test.js

```diff
@@ -14,7 +14,7 @@ const { ConfigFS } = require('../../../sdk/config_fs');
 const { TMP_PATH, set_nc_config_dir_in_config } = require('../../system_tests/test_utils');
 const { TYPES, ACTIONS } = require('../../../manage_nsfs/manage_nsfs_constants');
 
-const tmp_fs_path = path.join(TMP_PATH, 'test_nc_nsfs_account_cli');
+const tmp_fs_path = path.join(TMP_PATH, 'test_nc_account_cli');
```

Comment:
same here, it was just copied from the test account cli

Reply:
I'll change it, thanks.